rocketpy/environment/environment.py

```diff
@@ -1605,6 +1617,25 @@ def process_windy_atmosphere(self, model="ECMWF"):
             outputs="Pressure (Pa)",
             interpolation="linear",
         )
+        # Linearly extrapolate pressure to ground level
+        columns = data_array[:, (0, 1)]
+        pressure_ground = np.array(
+            [
+                [
+                    columns[1][0]
+                    + (-columns[1][1] / (columns[0][1] - columns[1][1]))
+                    * (columns[0][0] - columns[1][0]),
+                    0,
+                ]
+            ]
+        )
```

Comment:
I don't have much experience to talk about the validity of this extrapolation, but I advise being cautious with it so that things do not lose physical meaning. If necessary, we can find better ways to do it.

As a sidenote, if these code lines are so important, I would make a function in `tools.py` (or a static method here) to acomplish the extrapolation. I also recommend using more intermediate variables for some columns names so the code is easier to understand.

An extra would be also a simple unit test for this extrapolation method.